pkg/cmd/monitor_command/run_monitor_command.go

```diff
@@ -151,15 +189,21 @@ func (opt *RunMonitorOptions) Run() error {
 			return err
 		}
 
+		if err = sampler.TearDownInClusterMonitors(restConfig); err != nil {
```

Comment:
Also wire the teardown to a ctrl+C handler.

Reply:
Done